gridstatus/tests/test_spp.py

```diff
@@ -287,7 +286,7 @@ def test_get_lmp_real_time_weis_1_hour_range(self):
         ).normalize() - pd.Timedelta(
             days=1,
         )  # noqa
-        yesterday_1am = yesterday + pd.Timedelta(minutes=13)
```

Comment:
I'm not sure if there was a specific reason for this to be 13 minutes instead of 15.

Reply:
i think typo